src/utils.ts

```diff
@@ -58,17 +62,26 @@ export async function time<R>(cb: () => R, times: number[]) {
 
 export function cleanseSeparators(str: string): string {
 	return str
+		.replace(/\[.*?\]|「.*?」|｢.*?｣|【.*?】/g, "")
 		.replace(/[._()[\]]/g, " ")
 		.replace(/\s+/g, " ")
 		.trim();
 }
 
-export function reformatTitleForSearching(name: string): string {
+export function reformatTitleForSearching(name: string, isVideo?: boolean): string {
 	const seasonMatch = name.match(SEASON_REGEX);
 	const movieMatch = name.match(MOVIE_REGEX);
 	const episodeMatch = name.match(EP_REGEX);
+	let animeQuery: string | undefined = undefined;
+	if (isVideo) {
+		const { title, altTitle, release } = name.match(ANIME_REGEX)?.groups ?? {};
+		if (title) {
+			animeQuery = altTitle && altTitle.length ? Math.random() > 0.5 ? title : altTitle : title;
+			animeQuery += ` ${release}`;
+		}
+	}
```

Comment:
can this be refactored into a helper function

Reply:
We could do this as extra searchees, downside is more load on trackers.

To do it at the searchee generation stage it would mean all anime searchees will have their searchee.name updated to have only title or altTitle. Which may actually be good since we can put the [tag] at the end as `-tag` and won't have to update release group parsing.

Otherwise we could handle it right when making queries but some functions would have to change to allow the potential of multiple and we would have to consider how to respect `delay`.

Reply:
I think handling it right when making queries makes the most sense to me. I'd be inclined not to respect `delay` for this case but could be swayed.

Reply:
I'm not fully sure this are the only changes necessary to support multiple queries from the same indexer.

Updated so that multiple queries are supported. Right now only anime uses it with altTitle if present but should be easily expandable in the future. If we want to add a delay between the queries we would need to update searchUrls in makeRequests to not be a flat map but a loop.

I also checked the value of adding altTitle to EP_REGEX with the dataset. Only about 1% of the anime matches contains an alternate title. So updating the non anime regex to support altTitle is not worth it.